python/samples/04.ai.c.actionMapping.lightBot/src/bot.py

```diff
@@ -114,16 +123,18 @@ async def on_lights_status(
     _context: ActionTurnContext[Dict[str, Any]],
     state: AppTurnState,
 ):
-    return "the lights are on" if state.conversation.lights_on else "the lights are off"
+    light_status = "the lights are on" if state.conversation.lights_on else "the lights are off"
+    logging.info(f"{light_status}")
+    return light_status
 
 
 @app.error
 async def on_error(context: TurnContext, error: Exception):
     # This check writes out errors to console log .vs. app insights.
     # NOTE: In production environment, you should consider logging this to Azure
     #       application insights.
-    print(f"\n [on_turn_error] unhandled error: {error}", file=sys.stderr)
+    logging.info(f"\n [on_turn_error] unhandled error: {error}, {sys.stderr}")
     traceback.print_exc()
 
     # Send a message to the user
-    await context.send_activity("The bot encountered an error or bug.")
+    # await context.send_activity("The bot encountered an error or bug.")
```

Comment:
forgot to remove commented out lines in file

Reply:
@BMS-geodev I think you accidentally uncommented instead of removing for latest updates